src/utilities/getIntrospectionQuery.js

```diff
@@ -166,6 +175,7 @@ export type IntrospectionScalarType = {|
   +kind: 'SCALAR',
   +name: string,
   +description?: ?string,
+  +specifiedByUrl: ?string,
```

Comment:
This is technically a breaking change: code that was using `IntrospectionScalarType` no longer type-checks unless `specifiedByUrl` is explicitly set.